core/services/wifi/wpa_supplicant.py

```diff
@@ -47,223 +49,228 @@ def run(self, target: Union[Tuple[str, int], str] = target) -> None:
         self.sock.settimeout(10)
         self.sock.connect(self.target)
 
-    def send_command(self, command: str) -> bytes:
+    def send_command(self, command: str, timeout: float) -> bytes:
```

Comment:
timeout type is not specified (years, ms, s, ns ?)

Reply:
Good one. Will improve the docstrings for this function.

Reply:
Done